package.json

```diff
@@ -41,37 +41,43 @@
   ],
   "devDependencies": {
     "@types/graphql": "^14.5.0",
-    "@types/jest": "^27.4.0",
-    "@types/node": "^17.0.16",
+    "@types/jest": "^27.4.1",
+    "@types/node": "^17.0.23",
     "@types/object-path": "^0.11.1",
     "@types/reflect-metadata": "^0.1.0",
-    "docz": "^2.3.1",
+    "docz": "^1",
```

Comment:
Wrong version

Reply:
oh I had to revert this. I bumped it to major 2 by mistake-docz 2 is not compatible with docz 1 config and I've never migrated the config.
Version 1 should work fine.